src/main/java/sg/ncl/MainController.java

```diff
@@ -2417,14 +2415,72 @@ public String admin(Model model, HttpSession session) {
             datasetsList.add(dataset);
         }
 
+
         model.addAttribute("teamsMap", teamManager2.getTeamMap());
         model.addAttribute("pendingApprovalTeamsList", pendingApprovalTeamsList);
         model.addAttribute("usersList", usersList);
         model.addAttribute("userToTeamMap", userToTeamMap);
         model.addAttribute("dataList", datasetsList);
+
+        return "admin2";
+    }
+
+    @RequestMapping("/admin/experiments")
+    public String exp_dashboard(Model model, HttpSession session) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)